packages/account/src/Sections/Security/Passkeys/components/__tests__/tips-block.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { TipsBlock } from '../tips-block';
+
+describe('TipsBlock', () => {
+    const description_1 = 'Enable screen lock on your device.';
+    const description_2 = 'Sign in to your Google or iCloud account.';
+    const description_3 = 'Enable Bluetooth.';
```

Comment:
```suggestion
    const enable_screen_lock_tip = 'Enable screen lock on your device.';
    const google_sign_in_tip = 'Sign in to your Google or iCloud account.';
    const bluetooth_tip = 'Enable Bluetooth.';
```
I personally prefer semantic naming even in tests. Although it's up to you :)